pkg/kubectl/cmd/version.go

```diff
@@ -138,7 +138,11 @@ func (o *VersionOptions) Complete(cmd *cobra.Command) error {
 	return nil
 }
 
-func (o *VersionOptions) Validate() error {
+func (o *VersionOptions) Validate(args []string) error {
+	if len(args) > 0 {
+		return fmt.Errorf("not expect too many arguments")
```

Comment:
`unexpected arguments`

What do other commands say?

Reply:
Actually, fewer `kubectl` command to validate the num of parameters. I found that `create.go` using `Unexpected args`:
```shell
if len(args) != 0 {
		return cmdutil.UsageErrorf(cmd, "Unexpected args: %v", args)
	}
```

Reply:
"unexpected arguments" reads better.

Also, `Complete` is where we take the CLI flags and transform them into a runnable struct, so I'd expect this failure there.

Reply:
Done， Thank for all review